utb/models.py

```diff
@@ -8,18 +8,36 @@ class User(models.Model):
     email = models.CharField(max_length=30, unique=True)
     n_reports = models.IntegerField(default=0)
 
+    def as_dict(self):
```

Comment:
Have you tried `self.__dict__.keys()`?

Reply:
https://stackoverflow.com/questions/109087/how-to-get-instance-variables-in-python

Reply:
The problem with that is that I don't want every field of the class in the dict